sdks/python/container/boot.go

```diff
@@ -409,7 +410,40 @@ func installSetupPackages(ctx context.Context, logger *tools.Logger, files []str
 	if err := pipInstallPackage(ctx, logger, files, workDir, workflowFile, false, true, nil); err != nil {
 		return fmt.Errorf("failed to install workflow: %v", err)
 	}
+	if err := logRuntimeDependencies(ctx, logger); err != nil {
+		logger.Warnf(ctx, "couldn't fetch the runtime dependencies: %v", err)
+	}
+	if err := logSubmissionEnvDependencies(ctx, logger, workDir); err != nil {
+		logger.Warnf(ctx, "couldn't fetch the submission environment dependencies: %v", err)
+	}
+
+	return nil
+}
 
+func logSubmissionEnvDependencies(ctx context.Context, logger *tools.Logger, dir string) error {
+	logger.Log(ctx, fnexecution_v1.LogEntry_Severity_DEBUG, "Logging submission environment dependencies:")
```

Comment:
To log at debug over the logging API you can invoke logger.Printf(), debug is the default.

Reply:
done